### PR TITLE
Fix lint failures

### PR DESCRIPTION
**Pull request by ashmeenkaur**
### Description
This CL fixes some of the lint failures we see today on the master branch.

### Link to the issue in case of a bug fix.
NA

### Testing details
1. Manual - NA
2. Unit tests - NA
3. Integration tests - NA

### CODE REVIEW

File: benchmarks/write_locally/main.go [hunk 1]
```diff
@@ -95,11 +94,6 @@ func run() (err error) {
 		// Overwrite.
 		var n int64
 		for n < *fFileSize && time.Since(start) < *fDuration {
-			toWrite := *fFileSize - n
```
[the remark this answers is not in the document]
Comment (ashmeenkaur) [reply benchmarks/write_locally/main.go:98]: Hey, if you'll notice, we're just assigning toWrite variable some value but not using it anywhere. So, this piece of code was redundant.

File: internal/perf/cpu.go [hunk 2]
```diff
@@ -44,7 +43,10 @@ func HandleCPUProfileSignals() {
 		}()
 
 		// Profile.
-		pprof.StartCPUProfile(f)
+		err = pprof.StartCPUProfile(f)
+		if err != nil {
+			logger.Infof("StartCPUProfile failed: %v", err)
```
Comment (vadlakondaswetha) [inline internal/perf/cpu.go:48]: can we make it a warning or error log.

Comment (vadlakondaswetha) [reply internal/perf/cpu.go:48]: also does it make sense to return from here instead of executing further steps?

**Review by vadlakondaswetha — COMMENTED** [review]
minor comments. other changes looks good.

File: internal/perf/cpu.go (hunk 2 above)
Comment (ashmeenkaur) [reply internal/perf/cpu.go]: func StartCPUProfile returns an error if profiling is already enabled. So, I think we should not break the flow here. I've changed the log level to ERROR.

File: internal/perf/cpu.go [hunk 3]
```diff
@@ -44,7 +43,11 @@ func HandleCPUProfileSignals() {
 		}()
 
 		// Profile.
-		pprof.StartCPUProfile(f)
+		err = pprof.StartCPUProfile(f)
+		if err != nil {
+			errorLogger := logger.NewError("")
+			errorLogger.Printf("StartCPUProfile failed: %v", err)
```
Comment (vadlakondaswetha) [inline internal/perf/cpu.go:49]: you should create a method like this to log errors incase error is not available on available logger. https://github.com/GoogleCloudPlatform/gcsfuse/blob/master/internal/logger/logger.go#L138

Creating a newError logger for every log is not correct way to do.

File: internal/perf/cpu.go (hunk 2 above)
Comment (vadlakondaswetha) [reply internal/perf/cpu.go]: how does it make sense to run these 2 steps if profile start itself failed:
time.Sleep(duration)
pprof.StopCPUProfile()

File: internal/perf/cpu.go (hunk 3 above)
Comment (ashmeenkaur) [reply internal/perf/cpu.go]: Updated! Thanks!

File: internal/logger/logger.go [hunk 4]
```diff
@@ -33,6 +33,8 @@ const GCSFuseInBackgroundMode string = "GCSFUSE_IN_BACKGROUND_MODE"
 var (
 	defaultLoggerFactory *loggerFactory
 	defaultInfoLogger    *log.Logger
+
```
Comment (vadlakondaswetha) [inline internal/logger/logger.go:36]: nit: remove this extra line

**Review by vadlakondaswetha — APPROVED** [review]

**Review by Tulsishah — APPROVED** [review]

**Pull request merged**